klaas/plotting.py

```diff
@@ -226,69 +226,3 @@ def plot_feature_importances(model, feature_names, ax=None):
     ax.set_xlabel('Feature importances')
     ax.set_title('The {} most important features'.format(len(feature_names[:20])))
     ax.figure.tight_layout()
-
-
-def plot_binned_auc(
```

Comment:
I would not remove this function completely. Just remove it from the executable.